lib/util/http/host-rules.spec.ts

```diff
@@ -542,4 +543,21 @@ describe('util/http/host-rules', () => {
       username: undefined,
     });
   });
+
+  it('should remove unallowed header from request', () => {
```

Comment:
```suggestion
  it('should remove forbidden header from request', () => {
```

Or maybe we should use `blocked` instead. At least drop the `unallowed`.